tests/Gedmo/Mapping/Mock/Extension/Encoder/Mapping/Annotations.php

```diff
@@ -1,15 +1,13 @@
 <?php
 
-// file: vendor/Extension/Encoder/Mapping/Annotations.php
-
 namespace Gedmo\Tests\Mapping\Mock\Extension\Encoder\Mapping;
 
 use Doctrine\Common\Annotations\Annotation;
 
 /**
  * @Annotation
  */
-final class Encode extends Annotation
+final class Annotations extends Annotation
```

Comment:
Should we change the file name to `Encode` instead?  sounds better IMHO

Reply:
I agree. Done.